cmd/notation/sign.go

```diff
@@ -47,14 +47,16 @@ func signCommand(opts *signOpts) *cobra.Command {
 		Short: "Sign artifacts",
 		Long: `Sign artifacts
 
-Prerequisite: a signing key needs to be configured using the command "notation key".
```

Comment:
We recommend users to use `notation key` to configure keys before signing. In other words, the original Signing process should still be the recommended way for Signing. Should we rephrase this line to reflect this?

Reply:
@patrickzheng200  Good suggestion. I rephrased this note to `Note: a signing key needs to be configured using the command "notation key" or with specifying a key ID`.